framework/aster-frame/src/trap/handler.rs

```diff
@@ -64,7 +123,22 @@ extern "sysv64" fn trap_handler(f: &mut TrapFrame) {
             handle_virtual_exception(f, &ve_info);
             return;
         }
-        panic!("cannot handle kernel cpu fault now, information:{:#x?}", f);
+        #[cfg(feature = "intel_tdx")]
+        if f.trap_num as u16 == 14 {
+            let mut pt = KERNEL_PAGE_TABLE.get().unwrap().lock();
+            unsafe {
+                let _ = pt.map(
+                    x86::controlregs::cr2(),
+                    vaddr_to_paddr(x86::controlregs::cr2()).unwrap(),
+                    PageTableFlags::SHARED | PageTableFlags::PRESENT | PageTableFlags::WRITABLE,
```

Comment:
In the process of using `set_shared` function to configure the virtio address, we need to access the physical address, and the virtio high address has not yet been mapped. This can be handled through the `page fault` function in kernel.